bot/exts/evergreen/trivia_quiz.py

```diff
@@ -85,16 +291,17 @@ async def quiz_game(self, ctx: commands.Context, category: str = None) -> None:
             await ctx.send(embed=start_embed)  # send an embed with the rules
             await asyncio.sleep(1)
 
-        topic = self.questions[category]
-
         done_question = []
         hint_no = 0
-        answer = None
+        answers = None
+
         while self.game_status[ctx.channel.id]:
             # Exit quiz if number of questions for a round are already sent.
             if len(done_question) > self.question_limit and hint_no == 0:
                 await ctx.send("The round has ended.")
-                await self.declare_winner(ctx.channel, self.game_player_scores[ctx.channel.id])
+                await self.declare_winner(
+                    ctx.channel, self.game_player_scores[ctx.channel.id]
+                )
```

Comment:
Is there a reason this is split over 3 lines, the repo is configured to have a max line length of 120 so this just seems like default black formatting

Reply:
will be resolved in future commit, currently resolved it in my clone